src/utils/attachments.ts

```diff
@@ -0,0 +1,40 @@
+import { flatten } from "fp-ts/lib/Array";
+import { PrescriptionData } from "generated/backend/PrescriptionData";
+import { CreatedMessageWithContent } from "generated/io-api/CreatedMessageWithContent";
+import { toBarcode } from "./barcode";
+
+const MIME_TYPES = {
+  png: "image/png",
+  svg: "image/svg+xml"
+};
+
+const toBarcodeAttachments = async (name: string, value: string) => {
+  const errorOrBarcodes = await toBarcode(value);
+  return errorOrBarcodes.fold(
+    () => [],
+    barcodes => [
+      { name, content: barcodes.png, mime_type: MIME_TYPES.png },
+      { name, content: barcodes.svg, mime_type: MIME_TYPES.svg }
+    ]
+  );
+};
+
+export const getMessageWithAttachments = async (
```

Comment:
it's always better to export functions instead of consts, can we convert this into

`export async function getMessageWithAttachments() {... }` ?